content/post/stash-v2024.12.18/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Introducing Stash v2024.12.18
+date: "2024-12-18"
+weight: 10
+authors:
+- Md Anisur Rahman
+tags:
+- backup
+- cli
+- disaster-recovery
+- kubedump
+- kubernetes
+- mongodb
+- postgresql
+- restore
+- stash
+---
+
+We are pleased to announce the release of [Stash v2024.12.18](https://stash.run/docs/v2024.12.18/setup/), packed with new features. You can check out the full changelog [HERE](https://github.com/stashed/CHANGELOG/blob/master/releases/v2024.12.18/README.md). In this post, we'll highlight the changes done in this release.
+
+### New Features
+
+#### Introducing the `multiDumpArgs` Parameter for MySQL Backups
+
+We are excited to introduce the `multiDumpArgs` parameter, which is available for all MySQL versions during the backup process.
+
+**What is `multiDumpArgs`?**
+
+The `multiDumpArgs` parameter can be specified under the `spec.task.params` section in the `BackupConfiguration`. It allows you to provide arguments for multiple dump commands, which will be executed sequentially.
+
+**Why Use `multiDumpArgs`?**
+
+This feature is designed for use cases where you need to take database backups into different dump files, such as:
+
+- Backing up `table-schema` and `table-data` into separate files.
+
+- Customizing multiple dump operations to suit your specific needs.
+
+Using `multiDumpArgs`, you have to provide dump arguments by separating each command with the `$args` placeholder.
+
```

Comment:
You can specify multiple dump arguments for different `mysqldump` commands by separating each with `$args` using the `multiDumpArgs` parameter. Here is an example: